src/utils/convert.js

```diff
@@ -876,6 +899,13 @@ const renderIndex = legacyApp => {
 
     templateContext.REQUIRES = importLines.join('\n');
 
+    const searchOrCreates = findSearchOrCreates(legacyApp);
+    if (_.isEmpty(searchOrCreates)) {
+      templateContext.SEARCH_OR_CREATES = null;
```

Comment:
do we need to set this explicitly? It'll be `undefined` by default until it's set. Could just have `not _.isEmpty` as the only branch.

Reply:
Unfortunely, we can't do that because `_.template()` throws an exception when a variable is undfined in the template.

Reply:
ah, got it! Thanks for the explanation